src/app/accounts/update-account-custom-fields.ts

```diff
@@ -0,0 +1,51 @@
+import { getAccountsConfig } from "@config"
+
+import { CouldNotFindError } from "@domain/errors"
+import { NoAccountCustomFieldsError } from "@domain/accounts"
+
+import { AccountCustomFieldsRepository, AccountsRepository } from "@services/mongoose"
+
+const { customFields: customFieldsSchema } = getAccountsConfig()
+
+const defaultValues = customFieldsSchema.reduce((acc, val) => {
+  if (!val.unique && val.defaultValue !== undefined) {
+    acc[val.name] = val.defaultValue
+  }
+  return acc
+}, {} as { [k: string]: AccountCustomFieldValues })
+
+export const updateAccountCustomFields = async ({
+  accountId,
+  updatedByUserId,
+  customFields,
+}: {
+  accountId: AccountId
+  updatedByUserId: UserId
+  customFields: { [k: string]: AccountCustomFieldValues }
+}): Promise<AccountCustomFields | ApplicationError> => {
+  if (!customFieldsSchema || customFieldsSchema.length <= 0)
+    return new NoAccountCustomFieldsError()
```

Comment:
AFAICT this should never be called since the GQL that would access this is not included if they aren't configured. Correct?